SpO2 system improvements

YetheSamartaka opened the pull request:
- Removed unnecessary setVariable of SpO2 value(Also this will help people to better identify the cause of death)
- Improved clarity of the cause of death due to terminal SpO2 for logging and acex_killTracker feature
- Improved performance of fnc_handleAirway.sqf
- Optimized exitWith conditions
- Added Exclusion variable checks for other mods that could utilize it to prevent certain things from happening(Like whole SpO2 change, Occlusions, Obstructions or Pneumothorax)
- Added alive check for handlePuking(Sometimes dead units puked even though they should not)
- Added Occlusion repeat timer
- added SpO2 perfusion multiplier

YetheSamartaka committed:
handleBreathing improvements

- removed unnecessary setVariable(Also this will help people to better identify the cause of death)
- improved clarity of the cause of death for logging and acex_killTracker feature
- added SpO2 perfusion multiplier

YetheSamartaka committed:
further optimalizations

- optimized exitWith conditions
- added Exlusion variable checks for other mods that could utilise it to prevent certain things from happening(Like whole SpO2 change, Occlusions, Obstructions or Pneumothorax)
- Added alive check for handlePuking(Sometimes dead units puked even though they should not)
- Added Occlusion repeat timer

Tomcat-SG commented on addons/airway/functions/fnc_handleAirway.sqf:
```diff
@@ -17,14 +17,13 @@
 
 
 if (random(100) <= GVAR(probability_obstruction)) then {
-    if !(_unit getVariable [QGVAR(obstruction), false]) then {
```
```suggestion
    if !(_unit getVariable [QGVAR(obstruction), false]) then {
```

Tomcat-SG commented on addons/airway/functions/fnc_handleAirway.sqf:
```diff
@@ -17,14 +17,13 @@
 if (random(100) <= GVAR(probability_obstruction)) then {
-    if !(_unit getVariable [QGVAR(obstruction), false]) then {
         _unit setVariable [QGVAR(obstruction), true, true];
-    };
```
```suggestion
    };
```

Tomcat-SG committed:
smaller fixes and german translations

Tomcat-SG reviewed (approved):
LGTM

YetheSamartaka committed:
removed unnecessary if and comments

- Obstruction is only rolled when the patient goes unconscious, so when this function is called, the patient never has this variable set to true thus that "if" does not have to be there.
- comments were from very early versions of this function not relevant to today's system

YetheSamartaka committed:
Tabs to Spaces
